dataprofiler/tests/labelers/test_data_labelers.py

```diff
@@ -313,13 +313,13 @@ def test_set_labels(self, mock_open, mock_load_model):
         data_labeler._model.requires_zero_mapping = True
         data_labeler.set_labels(['a', 'b'])
         data_labeler._model.set_label_mapping.assert_called_with(
-            label_mapping={'a': 1, 'b': 2})
+            label_mapping=['a', 'b'])
 
         # test require pad false
         data_labeler._model.requires_zero_mapping = False
         data_labeler.set_labels(['a', 'b'])
         data_labeler._model.set_label_mapping.assert_called_with(
-            label_mapping={'a': 0, 'b': 1})
+            label_mapping=['a', 'b'])
```

Comment:
Can we still manually set the label mapping, if needed? My understanding is that we could still use a dict if desired (though it wouldn't be recommended).

Reply:
Technically, but you could also do it at `fit` using the labels `parameter` as in the readme